_posts/en/newsletters/2021-08-11-newsletter.md

```diff
@@ -0,0 +1,168 @@
+fidelity bonds in JoinMarket and includes our regular sections with the
+summary of a Bitcoin Core PR Review Club meeting, suggestions for
+preparing for taproot, announcements of releases and release candidates,
+and descriptions of notable changes to popular infrastructure projects.
+
+## News
+
+- **Implementation of fidelity bonds:** the [JoinMarket 0.9.0][]
+  implementation of [coinjoin][topic coinjoin] includes [support][jm notes]
+  for [fidelity bonds][fidelity bonds doc].  As previously described in
+  [Newsletter #57][], the bonds improve the sybil resistance of the
+  JoinMarket system, increasing the ability for coinjoin initiators
+  ("takers") to choose unique liquidity providers ("makers").  Within
+  days of release, [over 50 BTC][waxwing toot] (currently valued at over
+  $2 million USD) had been placed in timelocked fidelity bonds.
+
+    Although the specific implementation is unique to JoinMarket, the
+    overall design may be useful in other decentralized protocols built
+    on top of Bitcoin.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+[Prefer to use txindex if available for GetTransaction][review club #22383] is a PR by Jameson Lopp
+which improves the performance of `GetTransaction` (and, by extension, the `getrawtransaction` RPC
+for users) by utilizing the transaction index (txindex) when possible. This change fixes an unexpected
+performance loss in which a call to `getrawtransaction` on a txindex-enabled node is significantly
+slower when called with the hash of the block that includes the transaction. The review club
+evaluated the cause of this performance issue by comparing the steps to retrieve a transaction with
+and without txindex.
+
+{% include functions/details-list.md
+
+  q0="What are the different ways `GetTransaction` can retrieve a transaction from disk?"
+  a0="The transaction can be retrieved from the mempool (if unconfirmed), by retrieving the entire
+  block from disk and searching for the transaction, or by using txindex to fetch the transaction
+  from disk by itself."
+  a0link="https://bitcoincore.reviews/22383#l-33"
+
+  q1="Why do you think that performance is worse when the block hash is provided (when txindex is
+  enabled)?"
+  a1="Participants guessed that the bottleneck was in the deserialization of the block. Another
+  process unique to fetching the entire block - albeit less time-consuming - is a linear search
+  through the entire list of transactions."
+  a1link="https://bitcoincore.reviews/22383#l-42"
+
+  q2="If we are looking up the transaction by block hash, what are the steps? How much data
+  is deserialized?"
+  a2="We first use the block index to find out which file and byte offset to find the transaction.
```

Comment:
Should this be `s/the transaction/the block/`?

Reply:
Ooh yes, good catch!